### PR TITLE
Byline and date should be below social shares

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?
As part of the article refresh, bylines should go below social shares. 

## What is the value of this and can you measure success?
It is part of a series of changes with the aim to make an entire article page look better

## Does this affect other platforms - Amp, Apps, etc?
It does affect AMP as well

## Screenshots
BEFORE:
image

AFTER:
image

## Tested in CODE?
Nope

### CODE REVIEW

**Comment by gtrufitt** [comment]
Looks like a double border there? 
image

**Review by gtrufitt — APPROVED** [review]
Other than a couple of rouge borders, LGTM 👍

**Review by SiAdcock — APPROVED** [review]
Awesome stuff

File: static/src/stylesheets/module/content/_content.scss [hunk 1]
```diff
@@ -498,6 +458,46 @@
...
+    .byline-img {
+        @include circular;
+        position: relative;
+        width: gs-span(1);
+        height: 60px; //Intentionally off grid
+        margin: $gs-baseline/2 0;
+        overflow: hidden;
+        background-color: $neutral-7;
+
+        // fixes for webkit not properly scaling/clipping the child element (still broken on some android browsers...)
```
Comment (sndrs) [inline static/src/stylesheets/module/content/_content.scss]: is this still needed, do you think?

Comment (NataliaLKB) [inline static/src/stylesheets/module/content/_content.scss]: @sndrs It is three years old... I don't think its needed anymore. Will delete

**Comment by NataliaLKB** [comment]
@gtrufitt good spot! Another reminder that I should get glasses... All fixed :)

**Pull request merged**